tutorials/feature-selection-tribuo-v4.ipynb

```diff
@@ -0,0 +1,539 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bb95d7c0",
+   "metadata": {},
+   "source": [
+    "# Feature Selection\n",
+    "\n",
+    "When working with tabular machine learning datasets it is quite common to collect as many columns as possible, and extract as many features from those columns. This can lead to very large feature spaces, which can cause issues when fitting certain kinds of ML models, as they may use irrelevant or noisy features which lead to poor test time performance. Additionally large feature spaces can increase training times, while making it tricky to build interpretable models as there are too many features to reason over. For these reasons the field of feature selection developed, which aims to find the relevant subset of features for a given prediction task from a large set of possible features.\n",
```

Comment:
```suggestion
    "When working with tabular machine learning datasets it is quite common to collect as many columns as possible, and extract many features from those columns. This can lead to very large feature spaces, which can cause issues when fitting certain kinds of ML models, as they may use irrelevant or noisy features leading to poor test time performance. Additionally, large feature spaces can increase training times, and make it difficult to build interpretable models as there are too many features to reason over. For these reasons the field of feature selection developed, which aims to find the relevant subset of features for a given prediction task from a large set of possible features.\n",
```